src/target/riscv/riscv-013.c

```diff
@@ -525,6 +588,19 @@ static int dmi_write(struct target *target, uint16_t address, uint64_t value)
 	return ERROR_OK;
 }
 
+int dmstatus_read(struct target *target, uint32_t *dmstatus,
+		bool authenticated)
+{
+	*dmstatus = dmi_read(target, DMI_DMSTATUS);
+	if (authenticated && !get_field(*dmstatus, DMI_DMSTATUS_AUTHENTICATED)) {
+		LOG_ERROR("Debugger is not authenticated to target Debug Module. "
+				"(dmstatus=0x%x). Use `riscv authdata_read` and "
```

Comment:
is it `riscv_authdata_read` or `riscv authdata_read` ? The comment mixes the two so seems likely that one is wrong

Reply:
Fixed.